lib/utils.re

```diff
@@ -79,47 +73,37 @@ let getClassesFromSelector = selector => {
 };
 
 /** Parses out the valid class names from the given CSS */
-let getAcceptableClassNames = css => {
-  // See if we've "cached" the acceptable names before
-  switch (acceptableNames^) {
-  | Some(names) => names
-  | None =>
-    let (cssRules, _) = parseStylesheet(css);
-
-    let rec gatherClassSelector = (existingClassNames, rule) => {
-      switch (rule) {
-      | Rule.Style_rule(styleRule) =>
-        let prelude = fst(styleRule.prelude);
-        switch (prelude) {
-        | [
-            (Component_value.Delim("."), _),
-            (Component_value.Ident(_), _),
-            ..._,
-          ] =>
-          List.fold_left(
-            (classNames, classNameFromSelector) =>
-              [unescapeIdent(classNameFromSelector), ...classNames],
-            existingClassNames,
-            getClassesFromSelector(prelude),
-          )
-        | _ => existingClassNames // Ignore other preludes
-        };
-      | Rule.At_rule({At_rule.name: ("media", _), At_rule.block, _}) =>
-        switch (block) {
-        | Stylesheet((rules, _)) =>
-          List.fold_left(gatherClassSelector, existingClassNames, rules)
-        | _ => []
-        }
-      // Ignore other prelude
-      | _ => existingClassNames
+let getAcceptableClassNames = (cssRules, css) => {
```

Comment:
Doesn't look like the `css` argument is used anymore

```suggestion
let getAcceptableClassNames = cssRules => {
```

Reply:
Indeed, thx!